tests/unit/modules/test_github.py

```diff
@@ -1,13 +1,224 @@
+from unittest import mock
+from unittest import TestCase
+
 import pytest
+import salt.modules.config as config_module
 import salt.modules.test as testmod
 import saltext.github.modules.github as github_module
+from salt.exceptions import CommandExecutionError
 
 
 @pytest.fixture
 def configure_loader_modules():
     module_globals = {
-        "__salt__": {"test.echo": testmod.echo},
+        "__salt__": {"test.echo": testmod.echo, "config.option": config_module.option},
+        "__opts__": {"mock": {"ruleset_type": "mocked_value"}},
```

Comment:
**issue (blocking):**  I think the "mock" option has special meaning in the state system. better to pick a different string